src/GovUk.Education.ExploreEducationStatistics.Publisher.Tests/Services/PublishingCompletionServiceTests.cs

```diff
@@ -26,6 +26,11 @@ public class PublishingCompletionServiceTests
     private readonly DataSetPublishingServiceMockBuilder _dataSetPublishingService = new();
     private readonly PublisherEventRaiserMockBuilder _publisherEventRaiser = new();
 
+    /// <summary>
+    /// Ensure that GetSUT is called immediate before the ACT phase.
+    /// This is because the _contentDbContext builder disposes the dbContext used during ARRANGE
+    /// and creates a fresh copy to ensure that any queries performed rehydrate the entities properly.
+    /// </summary>
```

Comment:
Cool, this seems like a good approach. Are we going to need this reminder on every test class though where we use ContentDbContextMockBuilder?